src/styles/blocks/page.scss

```diff
@@ -0,0 +1,5 @@
+.page {
+  &__content {
+    margin: 0;
+  }
+}
```

Comment:
you can rename this file on "reset.scss", because this is 
file resets the default settings and don't use elements classes in reset.css file
couch told me use tag selectors here (body,p, ul, for example, or any else, that you need)
don't use elements classes in reset.css file